pkgs/development/ocaml-modules/ffmpeg/ffmpeg-avdevice.nix

```diff
@@ -11,7 +19,8 @@ buildDunePackage {
   inherit (ffmpeg-base) version src useDune2;
 
   nativeBuildInputs = [ pkg-config ];
-  buildInputs = [ dune-configurator ];
+  buildInputs = [ dune-configurator ]
+    ++ lib.optionals stdenv.isDarwin [ AppKit AudioToolbox Cocoa CoreImage ForceFeedback OpenGL VideoToolbox ];
```

Comment:
I am unsure what is best split list or not ? ++ on this line or the line before

Reply:
Both are perfect for me.